README.md

```diff
@@ -108,16 +108,25 @@ opened where you can further experiment with the test.
 * `page.driver.scroll_to(left, top)` Scroll to a given position.
 * `element.send_keys(*keys)` Send keys to a given node.
 
+## Set user agent
+
+To change the user-agent of the driver for all following requests, use the `user-agent` browser option:
+
+```
+Capybara::Cuprite::Driver.new(app, { browser_options: { 'user-agent': 'Your own user agent string' } })
+```
+
+If you want to change the user-agent for one request, you can use the request headers.
 
 ## Request headers
 
 Manipulate HTTP request headers like a boss:
 
 ``` ruby
 page.driver.headers # => {}
-page.driver.headers = { "User-Agent" => "Cuprite" }
+page.driver.headers = { "Host" => "github.com" }
 page.driver.add_headers("Referer" => "https://example.com")
-page.driver.headers # => { "User-Agent" => "Cuprite", "Referer" => "https://example.com" }
+page.driver.headers # => { "User-Agent" => "Cuprite", "Host" => "https://github.com" }
```

Comment:
this line is quite correct because `add_headers` added `Referer` header one previous line, so it `page.driver.headers` should contain `Referer` header for sure, that's why it's a bit confusing.
Let's try this way: 
```diff
- page.driver.headers # => { "User-Agent" => "Cuprite", "Host" => "https://github.com" }
+ page.driver.headers # => { "Host" => "https://github.com", "Referer" => "https://example.com" }
```
Or even better to keep `User-Agent` example, like this:
```diff
- page.driver.headers = { "Host" => "github.com" }
+ page.driver.headers = { "User-Agent" => "Cuprite", "Host" => "https://github.com", }
```
```suggestion
page.driver.headers # => { "User-Agent" => "Cuprite", "Host" => "https://github.com", "Referer" => "https://example.com" }
```